packages/plugins/typescript-resolvers/src/root.handlebars

```diff
@@ -79,10 +79,10 @@ export interface IResolvers<TContext = {{{ getContext }}}> {
   {{#each scalars}}
     {{ convert name 'typeNames'}}{{#unless @root.config.strict}}?{{/unless}}: GraphQLScalarType;
   {{/each}}
-}
+} & { [typeName: string] : never };
```

Comment:
Are we sure it won't mark all properties as `never`?

Reply:
Yeah looks suspicious

Reply:
You can try it :)